ovn_cluster.sh

```diff
@@ -585,7 +623,7 @@ function start-chassis() {
     ovn_central=no
     ovn_remote=$1
     ovn_add_chassis=$2
-    start $ovn_central $ovn_remote $ovn_add_chassis
+    start $ovn_central $ovn_remote $ovn_add_chassis no
```

Comment:
what is that ```no``` for ?

Reply:
Removed it. In my initial attempt had thought of generating certificates in start() if "yes" is passed in $4.

Later I changed the approach to generate the ssl certificates during build itself. And I forgot to delete this.

Thanks for catching this.